installers/rpm-code/codes/ipykernel.sh

```diff
@@ -3,25 +3,25 @@
 ipykernel_main() {
     codes_dependencies common
     install_pip_install ipykernel
-    local i=3
-    local v=py$i
+}
+
+ipykernel_python_install() {
+    local -a e=()
+    for x in LD_LIBRARY_PATH PKG_CONFIG_PATH PYTHONPATH; do
```

Comment:
I don't think these values can come from the build environment. Maybe PKG_CONFIG_PATH. They normally are inherited via ~/.bashrc so we don't need to set them explicitly. Has something changed?

Reply:
Thanks, I see now how these come from home-env.

Does the jupyterhub process have the correct env? I think we need to use https://jupyterhub.readthedocs.io/en/stable/api/spawner.html#jupyterhub.spawner.Spawner.env_keep because [this is](https://github.com/jupyterhub/jupyterhub/blob/909b3ad4d708a7369343c8a8537c95c05aae3b67/jupyterhub/spawner.py#L433) only env that is passed to the notebook which doesn't contain `LD_LIBRARY_PATH`

Reply:
aaaeb0f